syft/pkg/cataloger/java/archive_parser.go

```diff
@@ -282,6 +302,89 @@ func artifactIDMatchesFilename(artifactID, fileName string) bool {
 	return strings.HasPrefix(artifactID, fileName) || strings.HasSuffix(fileName, artifactID)
 }
 
+func findPomLicenses(pomProjectObject *parsedPomProject) {
+	// If we don't have any licenses until now, and if we have a parent Pom, then we'll check the parent pom in maven central for licenses.
+	if pomProjectObject != nil && pomProjectObject.Parent != nil && len(pomProjectObject.Licenses) == 0 {
+		parentPom, err := getPomFromMavenCentral(pomProjectObject.Parent.GroupID, pomProjectObject.Parent.ArtifactID, pomProjectObject.Parent.Version)
+		if err != nil {
+			// We don't want to abort here as the parent pom might not exist in Maven Central, we'll just log the error
+			log.Debugf("unable to get parent pom from Maven central: %v", err)
```

Comment:
```suggestion
			log.Tracef("unable to get parent pom from Maven central: %v", err)
```
Since it's not clear if we should have been able to get the pom at all keeping this low in the logging calls makes sense.

Reply:
updated to trace